collector/collector.go

```diff
@@ -31,6 +32,8 @@ const (
 	// Query to check whether user/table/client stats are enabled.
 	userstatCheckQuery = `SHOW GLOBAL VARIABLES WHERE Variable_Name='userstat'
 		OR Variable_Name='userstat_running'`
+	// date layout
+	dateLayout = "Jan 02 15:04:05 2006 MST"
```

Comment:
Hmm, these values seem to have inconsistent timestamps.

I get this format as well:

```
Variable_name: Max_used_connections_time
        Value: 2021-06-01 01:39:39
```

Reply:
Thanks, I missed that one. it should handle the two formats better now.